connectors/protocol/connectors.py

```diff
@@ -858,7 +858,7 @@ def _create_object(self, doc_source):
             doc_source=doc_source,
         )
 
-    async def create(self, connector, trigger_method):
+    async def create(self, connector, trigger_method, job_type=JobType.FULL):
```

Comment:
Should we've a default value here? Could imagine that the caller code is more explicit, if we always need to specify the `job_type`.

Reply:
Make sense. Will update